cmd/runmqserver/qmgr.go

```diff
@@ -41,6 +41,38 @@ func createDirStructure() error {
 	return nil
 }
 
+// configureOwnership recursively handles ownership of files within the given filepath
+func configureOwnership(paths []string) error {
+	uid, gid, err := command.LookupMQM()
+	if err != nil {
+		return err
+	}
+	for _, root := range paths {
+		_, err = os.Stat(root)
+		if err != nil {
+			if os.IsNotExist(err) {
```

Comment:
Might be better to do a `continue` in case one of the directories doesn't exists. e.g. if the web server wasn't installed, then that directory wouldn't be there, but that's no reason not to continue.